doc/classes/Input.xml

```diff
@@ -47,8 +47,9 @@
 			<return type="Vector3">
 			</return>
 			<description>
-				If the device has an accelerometer, this will return the acceleration. Otherwise, it returns an empty [Vector3].
+				If the device has an accelerometer, this will return the acceleration. Otherwise, it will return [constant Vector3.ZERO].
```

Comment:
Just for consistency, we should avoid using the future tense. Technical writing is generally done all in the present tense. 

```suggestion
				Returns the acceleration of the device's accelerometer, if the device has one. Otherwise, the method returns [constant Vector3.ZERO].
```